EmailAuth/FirebaseEmailAuthUI/FUIEmailAuth.m

```diff
@@ -20,7 +20,11 @@
 #import <FirebaseCore/FIROptions.h>
 #import <FirebaseAuth/FIRAuth.h>
 #import <FirebaseAuth/FirebaseAuth.h>
+#if SWIFT_PACKAGE
```

Comment:
This `#if` is no longer necessary and `#import <GoogleUtilities/GULUserDefaults.h>` should work for both SPM and CocoaPods.